src/main/java/com/google/sps/servlets/articlesServlet.java

```diff
@@ -95,10 +128,44 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     }
 
     String articles = strBuf.toString();
+
+    Query query = new Query("Article");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
```

Comment:
Add comments to improve readability.

Reply:
added a new comment on latest pull request :)